packages/nova-react-test-utils/src/shared/shared-utils.ts

```diff
@@ -4,6 +4,8 @@ import type { makeDecorator } from "@storybook/preview-api";
 
 export type Variant = "apollo" | "relay";
 
+export type Environment = "test" | "storybook";
```

Comment:
Is it confusing to call this `Environment`?  I'm not sure exactly what would be better though.

It's perhaps fine as it's not really used directly.

Reply:
How about TestingEnvironmentType?

Reply:
Did change to something more explicit